projects/packages/search/src/instant-search/components/search-results.scss

```
@@ -24,6 +24,28 @@ $modal-max-width-lg: 95%;
 		color: $color-text;
 		background: $color-mark-background-default;
 	}
+
+	#jetpack-instant-search__overlay-focus-anchor {
+		position: absolute;
+		bottom: 0;
+		right: 0;
+		border: none;
+		background-color: inherit;
+		color: $color-text-light;
```

```suggestion
		color: inherit;
```
For dark theme, the text is not very visible, so probably just inherit here?

---

Nice catch. I didn't test this in Dark Mode so I overlooked the styling there. Will get this fixed up.